web-registry/src/components/organisms/MediaForm.tsx

```diff
@@ -56,80 +58,105 @@ function getURLListInitialValue(value?: urlList): urlList {
-  },
-  description: {
-    marginBottom: 0,
-    fontSize: theme.typography.pxToRem(16),
-  },
-  field: {
-    [theme.breakpoints.up('sm')]: {
-      marginBottom: theme.spacing(12),
-    },
-    [theme.breakpoints.down('sm')]: {
-      marginBottom: theme.spacing(10),
-    },
-  },
-  error: {
-    color: theme.palette.error.main,
-    borderColor: theme.palette.error.main,
-    marginTop: theme.spacing(1),
-    marginBottom: 0,
-    fontFamily: '"Lato",-apple-system,sans-serif',
-    fontWeight: 'bold',
-    [theme.breakpoints.up('sm')]: {
-      fontSize: theme.spacing(3.5),
-    },
-    [theme.breakpoints.down('sm')]: {
-      fontSize: theme.spacing(3),
-    },
-  },
   fullSizeMedia: {
     width: '100%',
+    marginTop: theme.spacing(2),
     [theme.breakpoints.up('sm')]: {
       height: theme.typography.pxToRem(290),
     },
     [theme.breakpoints.down('sm')]: {
       height: theme.typography.pxToRem(210),
     },
   },
-  galleryImage: {
+  customMarginTop: {
     [theme.breakpoints.up('sm')]: {
-      height: theme.typography.pxToRem(169),
-      flex: 1,
+      marginTop: theme.spacing(10),
     },
     [theme.breakpoints.down('sm')]: {
-      height: theme.typography.pxToRem(139),
+      marginTop: theme.spacing(8),
     },
   },
-  centerImages: {
-    display: 'flex',
-    justifyContent: 'space-between',
-  },
-  centerSmall: {
-    maxHeight: theme.typography.pxToRem(72),
-  },
-  smallButton: {
-    fontSize: theme.typography.pxToRem(14),
-    padding: theme.spacing(2, 3),
-  },
 }));
 
-const MediaForm: React.FC<MediaFormProps> = ({ submit, initialValues }) => {
+const projectShapeIri = getProjectShapeIri('C01');
+
+const MediaForm = ({ submit, initialValues }: MediaFormProps): JSX.Element => {
   const styles = useStyles();
-  const theme = useTheme();
   const apiUri = getApiUri();
   const { projectId } = useParams();
   const { confirmSave, isEdit } = useProjectEditContext();
-  const isTabletOrLarger = useMediaQuery(theme.breakpoints.up('sm'));
   const cropAspect = { aspect: 322 / 211 }; // px values pulled from mockups (width / height)
   const { data: graphData } = useShaclGraphByUriQuery({
     variables: {
-      uri: 'http://regen.network/ProjectPageShape',
+      uri: projectShapeIri,
     },
   });
 
+  const handleValidate = async (
+    values: MediaValues,
+  ): Promise<FormikErrors<MediaValues>> => {
+    const errors: FormikErrors<MediaValues> = {};
+    if (graphData?.shaclGraphByUri?.graph) {
+      const projectPageData = { ...getProjectPageBaseData(), ...values };
+      const report = await validate(
+        graphData.shaclGraphByUri.graph,
+        projectPageData,
+        projectShapeIri,
+      );
+      for (const result of report.results) {
+        const path: string = result.path.value;
+        const compactedPath = getCompactedPath(path) as
+          | keyof MediaValues
+          | undefined;
+        if (compactedPath) {
+          errors[compactedPath] = requiredMessage;
+        }
```

Comment:
let's keep the old validation logic
it doesn't matter so much now since everything is optional but as soon as this changes, the current code might not work
for instance, for `regen:previewPhoto` we need to set the error within the `'@value'` so it displays in the UI (since the field name  is `regen:previewPhoto.@value`)